projects/mercury/src/constants.js

```diff
@@ -9,6 +9,7 @@ export const MAIN_CONTENT_WIDTH = '65%';
 export const SIDE_PANEL_WIDTH = '35%';
 export const MAIN_CONTENT_MAX_HEIGHT = 'calc(100vh - 156px)';
 export const COLLECTIONS_PATH = 'collections';
+export const DEPARTMENTS_PATH = 'departments';
```

Comment:
Same as above, could it be configurable?

Reply:
I will make a 'ROOT_PATH'